docs/threat_model/threat_model.rst

```diff
@@ -392,7 +392,7 @@ What Compromise of the Admin's Property Can Surrender
       alerts.
    -  Access the credentials for the account the encrypt alerts are sent
       to.
-   -  Access the admin's personal GPG key.
+   -  Access the admin's personal GPG public key, if stored there.
```

Comment:
At least I'm assuming that we're talking about the public key. If we're not, we've not made this clear anywhere else in the docs (as far as I can tell, we make only one other passing reference to the admin's personal GPG key, in `passphrases.rst` above).